card/lib/card/format/render.rb

```diff
@@ -112,20 +112,13 @@ def pretty_path source_location
           source_location.second.to_s
       end
 
-      # setting (:alway, :never, :nested) designated in view definition
+      # :standard, :always, :never
       def view_cache_setting view
-        coded_setting = view_setting(:cache, view) || :standard
-        # method = self.class.view_cache_setting_method view
-        # coded_setting = respond_to?(method) ? send(method) : :standard
-        return :never if coded_setting == :never
-
-        # seems unwise to override a hard-coded "never"
-        (voo&.cache) || coded_setting
```

Comment:
I disagree with this comment.  Turning on view caching might be a smart way to stop performance problem in some cases.